### PR TITLE
open-cluster-management/backlog#5692 Devan rev

### PR DESCRIPTION
**Pull request by cadawson**

### CODE REVIEW

File: release_notes/known_issues.adoc [hunk 1]
```diff
@@ -239,15 +239,27 @@ Search maps RBAC for resources in the hub cluster. Depending on user RBAC settin
...
+
+. Open the `clusterdeployment` information for the managed cluster that is being removed by entering the following command:
++
+----
+oc edit clusterdeployment/<mycluster>
+----
++
+Replace _mycluster_ with the name of the managed cluster that you are destroying.
+
+. Ensure that the `clusterdeployment` is stopped by editing the content to remove the `hive.openshift.io/deprovision` finalizer.
```
Comment (dgoodwin) [inline release_notes/known_issues.adoc]: Suggest just "Remove the hive.openshift.io/deprovision finalizer to forcefully stop trying to clean up the cluster resources in the cloud." (as we're not making sure anything is stopped here, we're giving up trying to cleanup)

File: release_notes/known_issues.adoc [hunk 2]
```diff
@@ -239,15 +239,27 @@ Search maps RBAC for resources in the hub cluster. Depending on user RBAC settin
...
+
+. Save your changes.
```
Comment (dgoodwin) [inline release_notes/known_issues.adoc]: Suggest "and verify the ClusterDeployment is gone."

File: release_notes/known_issues.adoc (hunk 1 above)
Comment (cadawson) [reply release_notes/known_issues.adoc]: Reworded this. Thanks!

File: release_notes/known_issues.adoc (hunk 2 above)
Comment (cadawson) [reply release_notes/known_issues.adoc]: Added this task.

File: release_notes/known_issues.adoc [hunk 3]
```diff
@@ -239,15 +239,27 @@ Search maps RBAC for resources in the hub cluster. Depending on user RBAC settin
 
 When you destroy a managed cluster, the status continues to display `Destroying` after one hour, and the cluster is not destroyed. To resolve this issue complete the following steps:
 
-. Manually ensure that all of the provider resources that are associated with the managed cluster are cleaned up. 
+. Manually ensure that there are no orphaned resources on your cloud, and that all of the provider resources that are associated with the managed cluster are cleaned up.
+
+. Open the `ClusterDeployment` information for the managed cluster that is being removed by entering the following command:
++
+----
+oc edit clusterdeployment/<mycluster>
```
Comment (chrisahl) [inline release_notes/known_issues.adoc]: We need to be sure the user is using the correct namespace.  So we should add the `-n <namespace>` parameter to this command

**Comment by cadawson** [comment]
@dgoodwin : Thank you for the quick review. I captured your suggestions in the PR. Let me know if there are additional changes that are required.

File: release_notes/known_issues.adoc (hunk 3 above)
Comment (cadawson) [reply release_notes/known_issues.adoc]: Welcome back, @chrisahl ! I added the namespace to the command.

**Comment by cadawson** [comment]
Thanks @chrisahl and @dgoodwin . I sent a review to you each again to verify the updates. Thanks!

**Review by chrisahl — APPROVED** [review]

**Comment by cadawson** [comment]
Updated with the required changes. Merging to ensure that it makes it into the release. Let me know if there are additional questions. Thanks!

**Pull request merged**